### DISCUSSION TITLE
Possibility to build with library evolution enabled

### DISCUSSION THREAD

**Discussion by abel3cl** [pointfreeco/swift-parsing#284] [Ideas]
Make it possible to build with library evolution turned on, but without promising library stability
Refs: 

 - https://pointfreecommunity.slack.com/archives/C04L57GCR99/p1675852942315029
 - https://github.com/pointfreeco/swift-composable-architecture/discussions/1442
 - https://github.com/pointfreeco/swift-composable-architecture/pull/1443

**Comment by mbrandonw** [comment]
Hi @abel3cl, we don't have time to look into this now, but if you wanted to submit a PR we would happily take a look at it.
